camkes/templates/component.simple.c

```diff
@@ -482,7 +482,13 @@ void camkes_make_simple(simple_t *simple) {
             /*# If there is no size specified in the configuration then we assume the cnode
                 will be as small as possible to hold all the capabilities that are currently
                 defined #*/
-            simple_data.cnodesizebits = CONFIG_WORD_SIZE - __builtin_clz(/*? holding_slot ?*/) + 1;
+#if CONFIG_WORD_SIZE == 32
+                simple_data.cnodesizebits = CONFIG_WORD_SIZE - __builtin_clzl(/*? holding_slot ?*/) + 1;
```

Comment:
If I understand the issue correctly, we are running into the issue that there is no dedicated `clz()` for `word_t`. I would suggest we define a `clz_word()` in `seL4/include/util.h` with the defines here, and then use this here. That is better for readability and fits nicer with the comment we have in `util.h` already about the clz functions.

Reply:
Sorry, ignore that. The include `seL4/include/util.h` is kernel internal only. We could provide this in `libsel4/include/sel4/macros.h`? Or just here at the top of the file to have a minimal change.

Reply:
I found some related works in `util_libs/libutils/include/utils/builtin.h`. It defines the normal CTZ() macros, but no for specified WORD SIZE. Maybe we can add the `CLZ_WORD()` there.

Reply:
Can you do the change? I've added the "Test with" in the first comment, so CI for this PR will pick up the other PR then.